.github/workflows/test.yml

```diff
@@ -443,13 +464,15 @@ jobs:
         run: |
           INSTANCE_ID=$(gcloud compute instances describe sync-checkpoint-${{ env.GITHUB_REF_SLUG_URL }}-${{ env.GITHUB_SHA_SHORT }} --zone ${{ env.ZONE }} --format='value(id)')
           echo "Using instance: $INSTANCE_ID"
+
+          CONTAINER_NAME=""
```

Comment:
```suggestion
         CONTAINER_NAME=""
```

Reply:
When I applied this suggestion my linter complained, so I reverted the suggestion.